ruby/Gemfile

```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 gemspec
+
+gem 'rake', '~> 13.1'
```

Comment:
Whilst this is an auto-correct that's come in recently, we should de-activate it. A few of us aren't keen on it and in the ruby community a few don't like the default nature of on.

We lose info in the gem metadata for rubygems this way. Which sucks, but it's the way it is